### ISSUE TITLE
Feature request: Make column configuration playlist specific

### ISSUE THREAD

**Issue by HumanGerm** [DeaDBeeF-Player/deadbeef#1931]
### Album context
Select/create one playlist, it's for several albums.  Add columns for album art, album artist, etc.; group by artist & album title, etc.

I like playing a full album, and like seeing it displayed in context.

### User selected compilation/Random playlist
Select/create another playlist, this one is for a random selection of unrelated tracks that I want to play at a party.  I want to be able to just see a list of tracks, so I remove Album Art & Groupings because album context from the first scenario isn't relevant.

### Above are not compatible
Returning to my Album context playlist after creating my random playlist, and all the Album formatting has gone.

My request is to move the playlist settings from a global context to per-playlist 

### Information about the software:
Deadbeef version: 0.7.2
OS: Ubuntu 17.04

I know that this is probably a long-shot!  Many thanks for your consideration.  This has become my player of choice, finally stopped using Foobar2000 under wine, so thank you for all of your good work!

Kindest regards,
Stewart

**Comment by 0pLuS0** [comment]
I agree 100%, because without the ability, then as I mentioned in my report, when you try to play Radio Stations, it trys to fill up the album art covers and screws it up...

So for now I have in /.config/deadbeef and deadbeef.radio and when I play I just rename deadbeef to deadbeef.orig and deadbeef.radio to deadbeef when I want to play the radio and vice versus to play just songs back from .orig...

THANKS

**Comment by Alexey-Yakovenko** [comment]
That's an important point made right there.
Perhaps it would be much easier to change the code, so that all radio tracks group together, ignoring their metadata.
(Although I heard from people they want to group playlists differently anyway, even without radio)

**Comment by 0pLuS0** [comment]
I think the ```IDEA``` here is to have individual playlist tabs...

Create one with Album Art
Creat one without Album Art
Create one with other various tags the others don't have etc...

Hopefully this now explains the idea better... :)
